docs/question.md

```diff
@@ -117,7 +117,7 @@ The `question.html` is a template used to render the question to the student. A
 
 <p>
   $F = $
-  <pl-number-input answers_name="F" comparison="sigfig" digits="2" /> $\rm m/s^2$
+  <pl-number-input answers_name="F" comparison="sigfig" digits="2"></pl-number-input> $\rm m/s^2$
```

Comment:
```suggestion
  <pl-number-input answers_name="F" comparison="sigfig" digits="2" label="$F =$" suffix="$\rm m/s^2$"></pl-number-input>
```

Reply:
Good suggestion - done in 25e4af3, I also removed the now-extraneous `$F =$` from the previous line.